Core/ExpectedException/ExpectedExceptionDocumentConverter.cs

```diff
@@ -26,9 +26,9 @@ public class ExpectedExceptionDocumentConverter : IDocumentConverter
     public async Task<SyntaxNode> Convert (Document document)
     {
       var semanticModel = await document.GetSemanticModelAsync()
-                          ?? throw new ArgumentException ($"Document {document.Name} does not support a semantic model.");
+                          ?? throw new ArgumentException ($"Document '{document.FilePath}' does not support a semantic model.");

```

Comment:
(multiple occurrences)

I do think it sounds better that way, what do you think?

```suggestion
                          ?? throw new ArgumentException ($"Document '{document.FilePath}' does not support providing a semantic model.");
```